.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Why are these files here?

Reply:
I accidentally branched off of develop originally then tried to change the branch to patch here. I may need to redo this. I'm not sure why bug_report and request_feature are here.